<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1000</width>
    <height>800</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QWidget" name="tbWidge" native="true">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>20</y>
      <width>111</width>
      <height>571</height>
     </rect>
    </property>
   </widget>
   <widget class="QWidget" name="dfWidge" native="true">
    <property name="geometry">
     <rect>
      <x>130</x>
      <y>20</y>
      <width>561</width>
      <height>571</height>
     </rect>
    </property>
   </widget>
   <widget class="QWidget" name="apWidge" native="true">
    <property name="geometry">
     <rect>
      <x>700</x>
      <y>20</y>
      <width>191</width>
      <height>201</height>
     </rect>
    </property>
   </widget>
   <widget class="QWidget" name="fmWidge" native="true">
    <property name="geometry">
     <rect>
      <x>700</x>
      <y>230</y>
      <width>191</width>
      <height>351</height>
     </rect>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1000</width>
     <height>22</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuSprite_Editor">
    <property name="title">
     <string>File</string>
    </property>
    <addaction name="actionNew"/>
    <addaction name="actionOpen"/>
    <addaction name="actionSave"/>
    <addaction name="actionSave_As"/>
    <addaction name="actionExport"/>
   </widget>
   <widget class="QMenu" name="menuFile">
    <property name="title">
     <string>Help</string>
    </property>
   </widget>
   <addaction name="menuSprite_Editor"/>
   <addaction name="menuFile"/>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
  <action name="actionNew">
   <property name="text">
    <string>New</string>
   </property>
  </action>
  <action name="actionOpen">
   <property name="text">
    <string>Open</string>
   </property>
  </action>
  <action name="actionSave">
   <property name="text">
    <string>Save</string>
   </property>
  </action>
  <action name="actionSave_As">
   <property name="text">
    <string>Save As</string>
   </property>
  </action>
  <action name="actionExport">
   <property name="text">
    <string>Export</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections/>
</ui>
